packages/automerge-repo/src/Repo.ts

```diff
@@ -505,6 +505,31 @@ export class Repo extends EventEmitter<RepoEvents> {
       return this.storageSubsystem.id()
     }
   }
+
+  /**
+   * Waits for Repo to finish write changes to disk.
+   * @deprecated
```

Comment:
You marked a new code as deprecated?

Reply:
I discussed it with @pvh. He said that his feeling is that `flush()` isn't quite the right API for external consumption but is internally useful. We agreed that it is ok to add it but I proposed to mark `flush` as deprecated to warn users that this API will probably change soon.

Reply:
Nice! Thank you for the replay.